### PR TITLE
globalState component added.

### PR DESCRIPTION
**Pull request by Scott-Guinn**
Use of globalState demo set up in CaloricHistory component.
 (probably worth walking everyone through how to use this come integration time, but not a priority yet.)
Cleaned up 'guidance' comments in index.js file - things like "DONT USE EXPRESS"

### CODE REVIEW

File: components/caloricHistory.jsx [hunk 1]
```diff
@@ -1,12 +1,21 @@
-import React from 'react';
+import React, {useContext} from 'react';
+import {Context} from './globalState.js';
 import styles from '../styles/Home.module.css';
 
 const CaloricHistory = () => {
+  const {count, dummyString} = useContext(Context);
```
Comment (taylorsmart) [inline components/caloricHistory.jsx:6]: You keeping these in as an example for our demo?

File: components/globalState.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    return x + 2;
+  };
+
+  return (
+    <Context.Provider value={{
+      count,
+      setCount,
+      dummyString,
+      addTwo,
+    }}
```
Comment (taylorsmart) [inline components/globalState.js:29]: Did you want to add userInfo and setUserInfo in here for the demo?

**Review by taylorsmart — COMMENTED** [review]
Just noting the examples so we know to remove them later:
1) Count
2) setCount
3) dummyString
4) addTwo

userInfo
setUser Info currently defined but not used.

File: components/caloricHistory.jsx (hunk 1 above)
Comment (Scott-Guinn) [reply components/caloricHistory.jsx:6]: yeah until people (aka me) get used to using it, I think it's helpful to have a demo to refer to. I shouldn't have deleted that dummy component from testing, but I did....

File: components/globalState.js (hunk 2 above)
Comment (Scott-Guinn) [reply components/globalState.js:29]: yeah got to sort out where that's happening, I believe it happens on validation of sessionID/signup/login so I was waiting to have those components from Oren/Ben

Comment (emmaknor) [reply components/globalState.js:29]: is there a benefit to exporting the component upon definition(line 3)? is this a matter of cleanliness & if so would you like the team to be exporting components in this manner?

**Review by emmaknor — APPROVED** [review]

**Pull request merged**